libs/kano-networking/Makefile

```diff
@@ -49,4 +53,11 @@ clean-debug:
 	@rm -f $(DEBUG_OUTDIR)/$(OBJS)
 	@rm -f $(DEBUG_OUTDIR)/$(SONAME)
 
-clean: clean-release clean-debug
+link: all  # for Mac OS
+	mkdir -p /usr/local/include/kano
+	ln -fs `pwd`/includes/kano/* /usr/local/include/kano
+	ln -fs `pwd`/release/$(SONAME) /usr/local/lib
+
+unlink: all  # for Mac OS
+	rm -rf /usr/local/include/kano/networking
```

Comment:
Similarly, these `rm` commands should be replaced with `find`:
`find /usr/local/include/kano/networking -xdev -type l -delete`
IE, only delete symlinks.